libkbfs/keybase_daemon_local.go

```diff
@@ -84,25 +87,33 @@ func (c diskFavoriteClient) Shutdown() {
 }
 
 type memoryFavoriteClient struct {
-	favorites map[string]keybase1.Folder
+	favorites map[keybase1.UID]map[string]keybase1.Folder
```

Comment:
The problem was that `keybase1.Folder` contains more fields than just the name + public, so we can't have a per-UID set of `Folder` objects here.